src/pages/reset-password/index.tsx

```diff
@@ -1,14 +1,35 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Button } from '@components/Button'
 import { Container } from '@components/Container'
 import { LeftIcon } from '@components/Icons'
 import Link from 'next/link'
+import { useRouter } from 'next/router'
 
 export const ResetPassword = () => {
   const error = false
+  const router = useRouter()
+  const [emailValue, setEmailValue] = useState<string>('')
+
+  const handleSend = () => {
+    const emailVerification: boolean = /\S+@\S+\.\S+/.test(emailValue)
+
+    if (!emailVerification || !emailValue.length) {
+      return
+    }
```

Comment:
in case of error let's put error message

Reply:
dane